### DISCUSSION TITLE
3/06/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#225] [Daily Sprint]
Que hice ayer
Que hago hoy
Impedimentos

**Comment by Pamela198713** [comment]
Estuve con algunas pantallas corrigiendo errores. Acomode repositorios y la wiki.
Hoy se seguirá con los detalles que falten.
El tiempo. Y quizás un poco de falta de organización

**Comment by rpgrca** [comment]
Ayer terminé con toda la lógica de la compra y la del contacto. Hoy terminé todo el look de la app que faltaba, voy a ponerme a testear a ver si hay revientes groseros y después arreglar el sitio web.

**Comment by fmdcba** [comment]
Se me paso la hora para escribir. En el día de ayer estuvimos afinando detalles de web y móvil para la entrega final y actualizando la documentación.
